Fixes #7170 - Updates Import-PowerShellDataFile (#7178)
MicrosoftDocs/PowerShell-Docs: reference/7.2/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md

Accept wildcard characters: True
```

### -SkipLimitCheck

> By default `Import-PowerShellDataFile` has a limit of 500 keys you can import from a `.psd1` file.
> The **SkipLimitCheck** switch will bypass this limit and allow you to import more keys.

Reviewer: ```suggestion
By default `Import-PowerShellDataFile`  imports only 500 keys  from a `.psd1` file.
Use **SkipLimitCheck** to import more then 500 keys. 
```
